skyportal/handlers/api/observingrun.py

```diff
@@ -102,60 +102,39 @@ def get(self, run_id=None):
                   schema: Error
         """
         if run_id is not None:
-            run = (
-                DBSession()
-                .query(ObservingRun)
-                .options(
-                    joinedload(ObservingRun.assignments)
-                    .joinedload(ClassicalAssignment.obj)
-                    .joinedload(Obj.thumbnails),
-                    joinedload(ObservingRun.assignments).joinedload(
-                        ClassicalAssignment.requester
-                    ),
-                    joinedload(ObservingRun.instrument).joinedload(
-                        Instrument.telescope
-                    ),
-                    joinedload(ObservingRun.assignments)
-                    .joinedload(ClassicalAssignment.obj)
-                    .joinedload(Obj.sources)
-                    .joinedload(Source.group),
-                )
-                .filter(ObservingRun.id == run_id)
-                .first()
-            )
-
-            if run is None:
-                return self.error(
-                    f"Could not load observing run {run_id}", data={"run_id": run_id}
-                )
+            # These are all read=public, including Objs
+            options = [
+                joinedload(ObservingRun.assignments)
```

Comment:
You'll probably need to something like `return_data = baselayer.app.model_util.recursive_to_dict(return_data)` in order for these loaded relationships to actually be returned, e.g. https://github.com/skyportal/skyportal/pull/1833/files#diff-0200612f282483e1172207de90a4db4788b3ca5ce55ab5105938d876e11eb2e0R731